go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/argoproj/argo-cd/v2
 
-go 1.19
+go 1.21
```

Comment:
why this is required ?

Reply:
@pasha-codefresh looks like `notifications-engine` upgraded go version due to a dependency requirement: https://github.com/argoproj/notifications-engine/commit/c7e26ba718f3b78f0576952384719d0b62cd700c

Reply:
This is breaking our builds on RHEL-8, where go 1.19 is the go version used.